README.md

```diff
@@ -679,6 +679,17 @@ Offline Installations   | ✓     | x   | x   | x    |
   they could find ways to do an offline install for any package manager, but only `gomod` supports
   this out of the box (i.e. the user does not need to change their workflow).
 
+#### Limitations
+
+Cachito collects all dependencies of specified package manager types from a repository
+regardless of whether they are used by a certain application or not.
+For example, if some repository contains global `go.mod` file, dependencies from the file would be
+used for all container images built from the repository (see [gomod](#gomod) for more details).
```

Comment:
```suggestion
For example, if a repository contains a global `go.mod` file, all dependencies from this file will be present in the resulting archive and will also be cached to allow the future reproduction of the same build (see [gomod](#gomod) for more details).
```

Reply:
I think this is more in line with what Cachito actually does, since the archive could be used for whatever reason (although we know it is not). WDYT?